src/components/GameCard/GameCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import styles from './GameCard.module.css'
+import PropTypes from 'prop-types'
+
+const GameCard = ({ title, img }) => {
+    return (
+        <div className={styles.card}>
+            <img src={img ?? 'https://via.placeholder.com/250x250'} />
```

Comment:
Suggest moving the default link to some file as a constant.

Reply:
unfortunately, we don't have any files for that so far. It'd be better to do when we already know the exact structure of the project :)